.github/workflows/build.yml

```diff
@@ -7,6 +7,28 @@ on:
     branches: [ develop, master ]
 
 jobs:
+  build_arm64-binary: 
+     runs-on: ubuntu-latest
+     strategy:
+      matrix:
+        java: [ 8, 11 ]
```

Comment:
`openjdk-8-jdk` is installed in Docker. Does the declaration here make sense?

Reply:
Not needed here so I have removed it.